vizro-ai/src/vizro_ai/chains/_llm_models.py

```diff
@@ -12,26 +12,10 @@
 
 # TODO constant of model inventory, can be converted to yaml and link to docs
 PREDEFINED_MODELS: List[Dict[str, any]] = [
-    {
-        "name": "gpt-3.5-turbo-0613",
-        "max_tokens": 4096,
-        "wrapper": ChatOpenAI,
-    },
-    {
-        "name": "gpt-4-0613",
-        "max_tokens": 8192,
-        "wrapper": ChatOpenAI,
-    },
-    {
-        "name": "gpt-3.5-turbo-1106",
-        "max_tokens": 16385,
-        "wrapper": ChatOpenAI,
-    },
-    {
-        "name": "gpt-4-1106-preview",
-        "max_tokens": 128000,
-        "wrapper": ChatOpenAI,
-    },
+    {"name": "gpt-3.5-turbo-0613", "max_tokens": 4096, "wrapper": ChatOpenAI},
+    {"name": "gpt-4-0613", "max_tokens": 8192, "wrapper": ChatOpenAI},
+    {"name": "gpt-3.5-turbo-1106", "max_tokens": 16385, "wrapper": ChatOpenAI},
+    {"name": "gpt-4-1106-preview", "max_tokens": 128000, "wrapper": ChatOpenAI},
```

Comment:
For this I would either undo the changes or create a separate `config` file where it would keep the `PREDEFINED_MODELS` in the expanded format.

Reply:
Yeah, no worries! I can just add the comma back. I didn't go through all of the hundreds of changes 😄 So there might be even some cases where we want to change it after merging this PR. In this case, we just have to add the comma back. Then it will be formatted correctly 👍

Reply:
Yeah, I know that all changes in this PR are caused as a result of adding or removing trailing commas. I just wanted to say that in this case, I would probably revert back the trailing comma after `"wrapper": ChatOpenAI` lines and keep original extended format.